client/src/components/NicknameSearch.vue

```diff
@@ -25,27 +24,30 @@
 
 <script>
 import Avatar from '@/components/Avatar.vue'
-
 export default {
   components: {
     Avatar
   },
-  data () {
-    return {}
-  },
-  props: ['results']
+  props: ['results'],
+  computed: {
+    errorMessage () {
+      if (!this.results || this.results[0].error) {
```

Comment:
For future use - try making sure that "this.results[0]" actually exists before using chaining.